### PR TITLE
[Fix](csv_reader) Add a session variable to control whether empty rows in CSV files are read as NULL values.

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

Previously, empty rows in CSV files were ignored by Doris.
Now, add a session variable to control whether empty rows in CSV files are read as NULL values.

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -1058,6 +1060,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_PARALLEL_RESULT_SINK, needForward = true, fuzzy = true)
     private boolean enableParallelResultSink = true;
 
+    @VariableMgr.VarAttr(name = READ_CSV_EMPTY_LINE_AS_NULL)
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1063]: Add description field.

Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1063]: And needForward should be true

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Review by AshinGau — APPROVED** [review]

**Comment by AshinGau** [comment]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**